### PR TITLE
ADMINAPI-1011 2.x MSSQL Deployment in Docker Scenarios

### PR DESCRIPTION
**Pull request by jleiva-gap**
Create MSSQL container for Admin and Security Databases
Create Single Tenant and multitenant docker files for binaries and development.

### CODE REVIEW

File: Docker/api.Dockerfile
Comment (stephenfuqua) [inline Docker/api.Dockerfile:30]: Are you sure you want to hard-code this with no ability to override with a build argument?

The version "latest" (default value, line 20) does not work of course. The actual version _must_ be supplied as a `--build-arg`.

File: Docker/api.mssql.Dockerfile
Comment (stephenfuqua) [inline Docker/api.mssql.Dockerfile:13]: Can you use `wget` instead of `curl` above? And thus not need to install it on line 8?

File: Docker/api.mssql.Dockerfile
Comment (stephenfuqua) [inline Docker/api.mssql.Dockerfile:14]: Suggest adding a line above so that it is a little easier to read the file.

File: Docker/dev.Dockerfile
Comment (stephenfuqua) [inline Docker/dev.Dockerfile:1]: For better clarity, it might be useful to rename this to `dev.pgsql.Dockerfile`.

File: docs/docker.md
Comment (stephenfuqua) [inline docs/docker.md:66]: Please add a hard line-wrap here for easier reading.

File: Docker/dev.Dockerfile
Comment (jleiva-gap) [reply Docker/dev.Dockerfile:1]: I'll also rename api.Dockerfile to be consistent with this change.

**Review by stephenfuqua — APPROVED** [review]
I'm approving from "code owner" standpoint, but not going to merge. Leave it up to the Tools team to do final review & merge.

**Review by DavidJGapCR — APPROVED** [review]

**Pull request merged**
